packages/diagrams/frontend/sirius-components-diagrams/src/renderer/node/FreeFormNode.tsx

```diff
@@ -43,7 +43,7 @@ const freeFormNodeStyle = (
     opacity: faded ? '0.4' : '',
     transform: rotation,
     ...style,
-    backgroundColor: getCSSColor(String(style.backgroundColor), theme),
+    backgroundColor: getCSSColor(String(style.backgroundColor ?? 'transparent'), theme),
```

Comment:
Why should we hardcode a specific value here? And why `transparent`? Shouldn't the end result of getCSSColor return `null` or `undefined` (I don't recall which one is used to retrieve the default value)?

Reply:
The issue is because `style.backgroundColor` returns `undefined`.
Converted to string that gives `"undefined"` which make the background color keeping its actual value.
In my opinion, since the value `undefined` is a valid value for MUI, `getCSSColor` should have this signature: `getCSSColor = (value: string | undefined, theme: Theme): string | undefined`.